dashboards/grafana/README.md

```diff
@@ -0,0 +1,180 @@
+# Grafana Dashboards
+
+This project contains updated Helidon grafana dashboards and the instructions on how to 
+configure and enable additional metrics for Helidon SE and Helidon MP.
+
+> Note: This document and it's contents are work in progress only and not complete. 
+> The information and dashboards may change or be removed.
+
+## Table of Contents
+
+* [Software Pre-requisites](#software-pre-requisites)
+* [References](#references)
+* [Helidon Setup](#helidon-setup)
+  + [Application Identification](#application-identification)
+  + [Required Dependencies](#required-dependencies)
+  + [Enabling Metrics](#enabling-metrics)
+* [Prometheus Setup](#prometheus-setup)
+* [Grafana Setup](#grafana-setup)
+* [grafana Dashboard Images](#grafana-dashboard-images)
+
+## Software Pre-requisites
+
+* JDK21 or JDK24
+* Grafana 10.0.13+
+
+## References
+
+See the following for detailed information in Helidon metrics:
+
+* [SE Metrics](https://helidon.io/docs/latest/se/guides/metrics#basic-and-extended-kpi)
+* [Web Client Metrics](https://helidon.io/docs/latest/se/guides/webclient#WebClient-Metrics#WebClient-Metrics)
+* [MP Metrics](https://helidon.io/docs/latest/mp/metrics/metrics)
+* [MP REST Metrics](https://helidon.io/docs/latest/mp/guides/metrics#controlling-rest-request-metrics)
+* [Helidon Labs Virtual Threads - JDK24](https://github.com/helidon-io/helidon-labs/tree/main/incubator/virtual-threads-metrics)
+
+## Helidon Setup
+
+### Application Identification
+
+To ensure we can identify individual helidon applications as well as allowing drill-through, you should add the
+following labels to your generated metrics using the system property or `application.yaml` (SE) or `microprofile-config.properties` (MP):
+
+**Helidon SE**
+
+* System Property: `-Dmetrics.tags="se_app=my-se-app"`
```

Comment:
SE metrics config supports `appName` (similar to MP's `mp.appName`). Using `tags` works as well, but we might as well use `appName` for clarity?

* System Property: `-Dmetrics.appName=my-se-app`
* `application.yaml`:
   ```yaml
   metrics:
     appName: "my-se-app"
   ```

Reply:
@tjquinno I did find an issue with this, let me figure out what it was and get back to you.

Reply:
Not sure if this is what you are seeing, but make sure you are using an up-to-date build of Helidon 4 (release 4.1.7 or later or the `main` branch). I found a problem in the handling of global tags and the app name which I fixed in February.

Reply:
Fixed now